app/views/home/_product.html.erb

```diff
@@ -0,0 +1,23 @@
+<% @products.each do |product| %>
```

Comment:
cần for each nữa ko
chú tìm hiểu lại render partial nào

Reply:
Dạ em ready ạ